jans-cedarling/cedarling/src/log/log_entry.rs

```diff
@@ -147,24 +147,24 @@ pub struct AuthorizationLogInfo {
 
 /// Person authorize info
 #[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
-pub struct PersonAuthorizeInfo {
+pub struct UserAuthorizeInfo {
     /// cedar-policy user/person principal
-    pub person_principal: String,
+    pub principal: String,
```

Comment:
you use 
```
 #[serde(flatten)]
    pub person_authorize_info: Option<UserAuthorizeInfo>,
    /// Workload authorize info
    #[serde(flatten)]
    pub workload_authorize_info: Option<WorkloadAuthorizeInfo>,
```

Do you understand that `UserAuthorizeInfo` and `WorkloadAuthorizeInfo` can't have same fields?

Reply:
I read the comment, but it didn’t make sense. A clearer way to put it would be:

> UserAuthorizeInfo and WorkloadAuthorizeInfo have the same structure but use different field names to avoid conflicts.

Anyway, I resolved the issue by using serde_rename in this commit: [6dd4ecc](https://github.com/JanssenProject/jans/pull/10436/commits/6dd4ecc543867c77492e7029bf186b2c755616aa).

Reply:
how about `diagnostics` ?